lib/matplotlib/tests/test_patches.py

```diff
@@ -808,3 +808,32 @@ def test_default_capstyle():
 def test_default_joinstyle():
     patch = Patch()
     assert patch.get_joinstyle() == 'miter'
+
+
+@image_comparison(["autoscale_arc"], extensions=['png', 'svg'],
+                  style="mpl20")
+def test_autoscale_arc():
+    fig, axs = plt.subplots(1, 3, squeeze=False, figsize=(4, 1))
```

Comment:
its a bit funny to squeeze false, and then just use axs[0]?

Reply:
Probably is. I think I just copied the code from somewhere and didn't really consider that parameter.

Reply:
Ahh, indeed a bit weird how only using axs[0] even worked...